cmake/CMakeLists.txt

```diff
@@ -1156,6 +1156,10 @@ if(onnxruntime_USE_OPENVINO)
   elseif (${VER} MATCHES "2021.3" OR $ENV{INTEL_OPENVINO_DIR} MATCHES "2021.3")
     set(OPENVINO_VERSION "2021.3")
     add_definitions(-DOPENVINO_2021_3=1)  
+  elseif ($ENV{INTEL_OPENVINO_DIR} MATCHES "openvino")
+    set(OPENVINO_VERSION "2021.3")
```

Comment:
can you explain this case?
why are both OPENVINO_2021_3 and OPENVINO_2021_4 set to 1?

Reply:
we still have not enable openvino 2021 4 so we need to enable 2021 3 operator to ensure that onnxruntime shows the same compatibility as 2021 3 for any binary built from source code. and the config param is capsulated in 2021_4 because this is eventually going to happen